bootstrap-extensions/pom.xml

```diff
@@ -52,13 +54,13 @@
         <dependency>
             <groupId>org.webjars</groupId>
             <artifactId>tempusdominus-bootstrap-4</artifactId>
-            <version>5.39.0</version>
+            <version>${tempusdominus.version}</version>
         </dependency>
 
         <dependency>
-            <groupId>org.webjars.bower</groupId>
+            <groupId>org.webjars</groupId>
             <artifactId>momentjs</artifactId>
-            <version>2.24.0</version>
```

Comment:
@martin-g this version reported to have vulnerability ....
I would update before release ....